src/main/java/com/cleanroommc/modularui/value/sync/PanelSyncManager.java

```diff
@@ -194,9 +201,6 @@ public IPanelHandler panel(String key, PanelSyncHandler.IPanelBuilder panelBuild
         if (sh != null) return (IPanelHandler) sh;
         PanelSyncHandler syncHandler = new PanelSyncHandler(panelBuilder, subPanel);
         this.subPanels.put(key, syncHandler);
-        if (isInitialised()) {
-            this.modularSyncManager.getMainPSM().registerPanelSyncHandler(key, syncHandler);
-        }
```

Comment:
???

Reply:
the psm is never initialized at this point, so `isInitialised()` always returns false

Reply:
it also gets registered later when the psm is initialized